### PR TITLE
add Objective-C example

### PR DESCRIPTION
**Pull request by LeOric32**
Signed-off-by: Dmitry <dmitryzhuk99@gmail.com>

### CODE REVIEW

File: map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+
+@interface ViewController ()
+@property (weak, nonatomic) IBOutlet NMAMapView *mapView;
+@property (strong, nonatomic) NMACoreRouter *router;
+@property (strong, nonatomic) NMARoute *route;
+@property (strong, nonatomic) NMAMapRoute *mapRoute;
+@property (strong, nonatomic) NMAGeoBoundingBox *geoBoundingBox;
+@property (strong, nonatomic) UIActivityIndicatorView *activityView;
+@property (strong, nonatomic) NMAMapDataPrefetcher *prefetcher;
+@property (nonatomic) NSUInteger radius;
```
Comment (dashchak) [inline map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m:16]: Make it as const not property

File: map-data-prefetcher-ios/MapDataPrefetcherApp.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+			};
+			name = Release;
+		};
+		9664534E26CD5AA800617D29 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
+				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = L7P3R4HARK;
```
Comment (dashchak) [inline map-data-prefetcher-ios/MapDataPrefetcherApp.xcodeproj/project.pbxproj:303]: Remove team

File: map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+    NSMutableArray* stops = [[NSMutableArray alloc] initWithCapacity:2];
+    
+    // START: 4350 Still Creek Dr
+    NMAGeoCoordinates* hereBurnaby =
+    [[NMAGeoCoordinates alloc] initWithLatitude:49.264178 longitude:-123.117542];
+    // END: Langley BC
+    NMAGeoCoordinates* langley =
+    [[NMAGeoCoordinates alloc] initWithLatitude:49.229185 longitude:-123.094052];
+    
+    [stops addObject:hereBurnaby];
```
Comment (dashchak) [inline map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m:58]: NSArray *stops = @[hereBurnaby, langley];

File: map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+    __weak __typeof__(self) weakSelf = self;
+    // Trigger the route calculation
+    [self.router
+     calculateRouteWithStops:stops
+     routingMode:routingMode
+     completionBlock:^( NMARouteResult* routeResult, NMARoutingError error ) {
+        // make strong self object for method calls in block
+        __typeof__(self) strongSelf = weakSelf;
+        if ( !error ) {
+            if ( routeResult && routeResult.routes.count >= 1 ) {
```
Comment (dashchak) [inline map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m:82]: > 0

File: map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+        [self showMessageWithTitle:@"Error"
+                           message:[NSString stringWithFormat:@"Can't estimate route. Error code: %d",
+                                    (int)error]];
+        return;
+    }
+    [self showRoute];
+}
+
+#pragma mark - NMAMapDataPrefetcherListener
+- (void)prefetcher:(NMAMapDataPrefetcher *)prefetcher didEstimate:(NSInteger)dataSizeKB forRequestId:(NSInteger)requestId withSuccess:(BOOL)success
```
Comment (dashchak) [inline map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m:178]: Please keep row length at max 100 chars

File: map-data-prefetcher-ios/MapDataPrefetcherApp.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -0,0 +1,357 @@
...
+			};
+			name = Debug;
+		};
+		9664534F26CD5AA800617D29 /* Release */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
+				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = L7P3R4HARK;
```
Comment (dashchak) [inline map-data-prefetcher-ios/MapDataPrefetcherApp.xcodeproj/project.pbxproj:321]: Remove team

File: map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m (hunk 1 above)
Comment (LeOric32) [reply map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m:16]: Done

File: map-data-prefetcher-ios/MapDataPrefetcherApp.xcodeproj/project.pbxproj (hunk 2 above)
Comment (LeOric32) [reply map-data-prefetcher-ios/MapDataPrefetcherApp.xcodeproj/project.pbxproj:303]: Done

File: map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m (hunk 3 above)
Comment (LeOric32) [reply map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m:58]: Done

File: map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m (hunk 4 above)
Comment (LeOric32) [reply map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m:82]: Done

File: map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m (hunk 5 above)
Comment (LeOric32) [reply map-data-prefetcher-ios/MapDataPrefetcherApp/ViewController.m:178]: Done

File: map-data-prefetcher-ios/MapDataPrefetcherApp.xcodeproj/project.pbxproj (hunk 6 above)
Comment (LeOric32) [reply map-data-prefetcher-ios/MapDataPrefetcherApp.xcodeproj/project.pbxproj:321]: Done

**Pull request closed without merging**